### ISSUE TITLE
Failing CI tests: wikibase extensions not reachable

### ISSUE THREAD

**Issue by dajuno** [MaRDI4NFDI/portal-compose#180]
**Issue description**:

Since today, CI tests fail for `main` et al.
This happens also locally!

To reproduce: 
```
docker volume prune
docker-compose pull
docker-compose up -d
./run_tests.sh
```
will result in
```
Tests that hit the GUI using Selenium
----------------------------------------
Green 2.5.1, Coverage 4.5.1, Python 3.7.2

test.EmbedVideoTest
  EmbedVideoTest
E   Check that extensions are listed in Version page
green.suite
  GreenTestSuite
E   Check that extensions are listed in Version page
...
```

- this happens with current HEAD of main
- also with older commits previously checked successfully (e.g., #21f692a)
- last successful CI run was on 31/03 at 6:43pm, https://github.com/MaRDI4NFDI/portal-compose/actions/runs/2072405255
- cause of the failure is probably be a docker image published in the mean time
 
**TODOS**: 

- find the cause of the failure
- locally build older versions of docker images that were changed yesterday and test

**Checklist for this issue**: 

- [ ] Assignee has been set for this issue 
- [ ] All fields of the issue have been filled 
- [ ] Issue has been assigned to the main project
- [ ] Code was merged
- [ ] Feature branch has been deleted and issues were updated / closed

**Comment by dajuno** [comment]
building docker-wikibase:main locally, with portal-compose:main, the tests fail as commented above. In addition, localhost:8080 gives me the error:
```
Fatal error: Uncaught ExtensionDependencyError: Nuke is not compatible with the current MediaWiki core (version 1.35.5), it requires: >= 1.35.6. in /var/www/html/includes/registration/ExtensionRegistry.php:407 Stack trace: #0 /var/www/html/includes/registration/ExtensionRegistry.php(231): ExtensionRegistry->readFromQueue(Array) #1 /var/www/html/includes/Setup.php(161): ExtensionRegistry->loadFromQueue() #2 /var/www/html/includes/WebStart.php(89): require_once('/var/www/html/i...') #3 /var/www/html/index.php(44): require('/var/www/html/i...') #4 {main} thrown in /var/www/html/includes/registration/ExtensionRegistry.php on line 407
```

**Comment by dajuno** [comment]
Indeed, MW 1.35.6 has been the dependency for Nuke for 2 weeks:
https://github.com/wikimedia/mediawiki-extensions-Nuke/commit/ba74bee1db32e914357a1586e8edef2cf18998f9

**Comment by dajuno** [comment]
Turns out mediawiki "LTS 1.35" was bumped to 1.35.6 (https://github.com/wikimedia/mediawiki-docker/commit/dc8413ba7e60a7e4d2c8a08987c1a6fdb68abaa9) but the docker image has not been updated (is at version 1.35.5).
I'll comment the Nuke extension temporally from docker-wikibase until the mediawiki image at dockerhub updated, check https://hub.docker.com/_/mediawiki?tab=tags&page=1&name=1.35

**Comment by dajuno** [comment]
temporarily fixed by #181 

**TODO: undo this when mediawiki image 1.35.6 was [published on dockerhub](https://hub.docker.com/_/mediawiki?tab=tags&page=1&name=1.35)**
